src/JuMP.jl

```diff
@@ -635,6 +635,37 @@ function time_limit_sec(model::Model)
     return MOI.get(model, MOI.TimeLimitSec())
 end
 
+"""
+    simplex_iterations(model::Model)
+
+Gets the cumulative number of simplex iterations during the optimization process after a call to `optimize!(model)`.
+Returns the total simplex iterations for all nodes in case of a mixed-integer program (MIP). Solver must implement
+MathOptInterface.SimplexIterations() to use this function.
+"""
+function simplex_iterations(model::Model)::Int64
```

Comment:
Two things:
- Instead of using `Int64`, you should generally use `Int`.
- Gurobi actually returns a `Float64` here to avoid overflow:
https://github.com/JuliaOpt/Gurobi.jl/blob/5822a697c02884f3310a0fc3f6a0e6d4a5942df1/src/MOI_wrapper.jl#L2089
 so it's not clear we want to have a return-type annotation.

I would remove.

Reply:
I agree with no return type. I actually had not specified a return type but then did so later, assuming it as good practice.